invenio_access/cli.py

```diff
@@ -119,14 +119,6 @@ def allow_action(action, argument):
     """Allow action."""
 
 
-@allow_action.command('any')
-def allow_any():
```

Comment:
I don't think so. We always assign permissions to specific users or have a permission factory which allow/forbids everybody.